protocol/v2/ssv/validator/task_unique_id.go

```diff
@@ -0,0 +1,14 @@
+package validator
+
+import (
+	"fmt"
+
+	spectypes "github.com/bloxapp/ssv-spec/types"
+
+	"github.com/bloxapp/ssv/protocol/v2/ssv/runner"
+)
+
+func getTaskUniqueID(runner runner.Runner, duty *spectypes.Duty) string {
+	epoch := runner.GetBaseRunner().BeaconNetwork.EstimatedEpochAtSlot(duty.Slot)
+	return fmt.Sprintf("<%v-e<%v>-s<%v>-v<%v>", duty.Type.String(), epoch, duty.Slot, duty.ValidatorIndex)
```

Comment:
please remove the `<` `>`

Reply:
and add a comment with an example of what the ID looks like

Reply:
and please write a unit test

Reply:
done